extension-manifest-v3/src/utils/tabs.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+export function openTabWithUrl(host, event) {
+  const { href } = event.currentTarget;
+  event.preventDefault();
+
+  Promise.resolve(chrome.tabs.create({ url: href })).then(() => {
```

Comment:
why `Promise.resolve`?

Reply:
In MV2 `chrome.tabs.create` does not return a promise (in our case Firefox target).

Reply:
It might not be required, as we shim `chrome` API in Firefox, using `browser.`. I'll check that.

Reply:
and how is it? :)

Reply:
It looks, like we can omit `Promise.resolve()`